src/components/scenes/CreateWalletSelectCryptoScene.js

```diff
@@ -100,11 +100,13 @@ export class CreateWalletSelectCryptoComponent extends React.Component<Props, St
     return item.walletType
   }
 
+  getItemLayout = (data: any, index: number) => ({ length: this.props.theme.rem(4.25), offset: this.props.theme.rem(4.25) * index, index })
```

Comment:
This is duplicated across multiple files would it be worth creating an exported function or hook like getCommonRowLayout?

Reply:
Good idea. I'll add a hook, but it should be named 'useRowLayout' since hooks starts with 'use'.